front/app/modules/commercial/id_vienna_saml/components/ViennaSamlButton.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { AUTH_PATH } from 'containers/App/constants';
+
+// i18n
+import messages from './messages';
+import { FormattedMessage } from 'utils/cl-intl';
+
+import { StyledAuthProviderButton } from 'components/SignUpIn/AuthProviders';
+import { TOnContinueFunction } from 'components/SignUpIn/AuthProviderButton';
+
+// typings
+import { TSignUpInFlow } from 'components/SignUpIn';
+
+interface Props {
+  flow: TSignUpInFlow;
+  onContinue: TOnContinueFunction;
+}
+
+const ViennaSamlButton = ({ onContinue, flow }: Props) => {
+  const setHref = () => {
+    window.location.href = `${AUTH_PATH}/vienna_citizen`;
```

Comment:
Why do you use different methods (`window.location.href`, `window.location.replace`) for redirect? It seems `window.location.replace` is preferrable https://stackoverflow.com/questions/42914666/react-router-external-link#comment115520136_44563899

Not a big deal though.

Reply:
But maybe let's be consistent with other code and always use `window.location.href`